### ISSUE TITLE
Validar compatibilidade com Java 7

### ISSUE THREAD

**Issue by viniciusls** [pjbank/pjbank-java-sdk#59]
* Executar testes em ambiente utilizando Java 7, visto que a SDK está sendo desenvolvida com base em Java 8 e muitos sistemas ainda utilizam a versão 7.

**Comment by msfidelis** [comment]
Precisamos avaliar também as mudanças que saíram no Java 9, e principalmente as coisas que podem ter sido depreciadas.

**Comment by viniciusls** [comment]
Sim, foi até o que eu te mandei no Slack mais cedo. Mas como o Java 9 foi lançado ontem e não estamos usando nada que tenha sido marcado como depreciado no 8, estamos seguros por algum tempo. A princípio cobrir o Java 7 é mais crucial.  Penso em colocar o Travis pra rodar nas 2 versões, assim os testes trarão uma boa garantia caso algo seja implementado e não esteja presente no 7.
